addons/binding/org.openhab.binding.unifi/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,31 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: UniFi Binding
+Bundle-SymbolicName: org.openhab.binding.unifi;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.3.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .
+Import-Package: 
+ com.fasterxml.jackson.annotation,
```

Comment:
Could you use GSON instead? See https://www.eclipse.org/smarthome/documentation/development/bindings/dependencies.html#for-json-processing

Reply:
I can, but this will require a bit of rework (and testing). I'll fork another branch for this and seek help from the community to test it.